### ISSUE TITLE
Creating a new CR for node-exporter

### ISSUE THREAD

**Issue by yp969803** [prometheus-operator/prometheus-operator#6380]
### What is missing? Please describe.

We are mising a CR for prometheus/node-exporter which will be helpful for scraping os level metrics, It should be used for scrapping node metrics using daemonsets and node-exporter

### Describe alternatives you've considered.

Manually placing daemonset in nodes and using node-exporter

**Comment by yp969803** [comment]
@ArthurSens @simonpasquier any reviews on this ?

**Comment by ArthurSens** [comment]
What benefits would this bring over just using a deamonset?

**Comment by yp969803** [comment]
User does not have to manually deploy the daemon set, it is automating the deployment of daemaonset and nodeexporter

**Comment by mviswanathsai** [comment]
From my understanding, deploying a node-exporter does not seem to be needing any configuration that is particular to `node-exporter` itself to be defined along with the pod/daemonset. Any config could be done at the Prometheus instance level, could this use case not be achieved with the existing *Monitor resources and/or the `ScrapeConfig` Alpha CRD?

**Comment by yp969803** [comment]
Yep it would be nice if we can configure node-exporter with current scapeConfigs CR

**Comment by slashpai** [comment]
kube-prometheus already does that for you https://github.com/prometheus-operator/kube-prometheus.

IMHO operator should be responsible for configuring prometheus only but since alertmanager is closely connected we have CR for that as well but including exporters to operator would be difficult to maintain in long run. It should have its own operator if ever there is a requirement otherwise kube-prometheus seems to be sufficient, you can use it as a library if you want to configure node-exporter only as well
